site/app/templates/submission/homework/SubmitBox.twig

```diff
@@ -141,7 +141,7 @@
     {# /Admin submission type selector #}
 
     {% if is_vcs %}
-        {% if vcs_host_type == 0 or vcs_host_type == 1 %}
+        {% if vcs_host_type == constant('app\\models\\gradeable\\GradeableUtils::VCS_TYPE_SUBMITTY_HOSTED') or vcs_host_type == constant('app\\models\\gradeable\\GradeableUtils::VCS_TYPE_SUBMITTY_HOSTED_URL') %}
```

Comment:
Can you make the `constant('PATH_TO_GradeableUtils...)` into a [macro](https://twig.symfony.com/doc/3.x/tags/macro.html) to make these lines easier to read? Something like:
```
{% macro vcsConst(vcs_type) %}
constant('app\\models\\gradeable\\GradeableUtils::' ~ vcs_type)
{% endmacro %}
```
at the end of the file should (I think) work to define a macro that can be called with `_self.vcsConst('VCS_TYPE_SUBMITTY_HOSTED')` or similar. Other than that this looks good to me.